om4labs/diags/moc/moc.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python3
+
+import argparse
+import io
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import m6plot
```

Comment:
fails in travis

Reply:
Once #13 gets merged, we can replace with `import om4labs.m6plot as m6plot`